wsapi.go

```diff
@@ -322,10 +322,10 @@ func (s *Session) heartbeat(wsConn *websocket.Conn, listening <-chan interface{}
 
 // UpdateStatusData ia provided to UpdateStatusComplex()
 type UpdateStatusData struct {
-	IdleSince *int   `json:"since"`
-	Game      *Game  `json:"game"`
-	AFK       bool   `json:"afk"`
-	Status    string `json:"status"`
+	IdleSince  *int    `json:"since"`
+	Activities *[]Game `json:"activities"`
```

Comment:
Probably should also be `[]*Game` to match `Presence.Activities`

Reply:
```suggestion
	Activities []*Game `json:"activities"`
```